docs/ops.html

```diff
@@ -477,16 +477,20 @@ <h4><a id="rep-throttle" href="#rep-throttle">Limiting Bandwidth Usage during Da
   Throttle was removed.</pre>
 
   <p>The administrator can also validate the assigned configs using the kafka-configs.sh. There are two pairs of throttle
-      configuration used to manage the throttling process. The throttle value itself. This is configured, at a broker
+      configuration used to manage the throttling process. This is configured, at a broker
```

Comment:
remove redundant sentence.

Reply:
This is not redundant. It refers to the two pairs of properties shown below.  You can say instead: 

```suggestion
      configuration used to manage the throttling process. First pair refers to the throttle value itself. This is configured, at a broker
```

then below you could amend the sentence to say: 
_There is also an enumerated set of throttled replicas:_ -> _Then there is the configuration pair of enumerated set of throttled replicas:_